src/main/include/log4cxx/helpers/serversocket.h

```diff
@@ -57,7 +57,7 @@ class LOG4CXX_EXPORT ServerSocket
 		*/
 		void setSoTimeout(int timeout);
 
-		static ServerSocketUniquePtr create(int port);
+		static ServerSocketUniquePtr create(int port, bool reuseAddress = true);
```

Comment:
This causes an ABI breakage. You need to add a new method that takes in two parameters.